client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -27,7 +27,18 @@ import { getPluginsForSite } from 'state/plugins/premium/selectors';
 
 class PurchasePlanDetails extends Component {
 	static propTypes = {
+		purchaseId: PropTypes.number,
```

Comment:
`isRequired`?

Reply:
No `purchaseId` effectively means "placeholder." This is an expected state.